Series id's with length less than seven chars breaks the adminui

```

at java.lang.Thread.run(Thread.java:829) [?:?]
Caused by: java.lang.StringIndexOutOfBoundsException: begin 0, end 7, length 6
at java.lang.String.checkBoundsBeginEnd(String.java:3319) ~\[?:?\]
at java.lang.String.substring(String.java:1874) ~\[?:?\]
at org.opencastproject.list.impl.ListProvidersServiceImpl.getList(ListProvidersServiceImpl.java:149) ~\[?:?\]
at org.opencastproject.adminui.endpoint.SeriesEndpoint.getUserSeriesByAccess(SeriesEndpoint.java:729) ~\[?:?\]
at org.opencastproject.adminui.endpoint.ListProvidersEndpoint.getFilters(ListProvidersEndpoint.java:251) ~\[?:?\]
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~\[?:?\]
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~\[?:?\]
at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~\[?:?\]
```
![image](https://user-images.githubusercontent.com/1658875/174967116-279e032e-1e70-4843-9c43-098e6a486ad4.png)


**To Reproduce**
Steps to reproduce the behavior:
1. Make a serie (via api) that has less than 7 chars in id
2. Click on any event
3. See error 

**Environment Information**
 - OC 11.7 
-  https://github.com/opencast/opencast/blob/0199014ff51735935a94c8b5ae54753537911065/modules/list-providers-service/src/main/java/org/opencastproject/list/impl/ListProvidersServiceImpl.java#L149
 - introduced here https://github.com/opencast/opencast/issues/3600

--- a/modules/list-providers-service/src/main/java/org/opencastproject/list/impl/ListProvidersServiceImpl.java
+++ b/modules/list-providers-service/src/main/java/org/opencastproject/list/impl/ListProvidersServiceImpl.java
@@ -144,9 +144,14 @@ public class ListProvidersServiceImpl implements ListProvidersService {
       for (Map.Entry<String,String> entry : list.entrySet()) {
         int repeated = Collections.frequency(list.values(), entry.getValue());
         if (repeated > 1) {
+          String newSeriesName = null;
           //If a series name is repeated, will add the first 7 characters of the series ID to the display name on the
           //admin-ui
-          String newSeriesName = entry.getValue() + " " + "(ID: " + entry.getKey().substring(0, 7) + "...)";
+          try {
+            newSeriesName = entry.getValue() + " " + "(ID: " + entry.getKey().substring(0, 7) + "...)";
+          } catch (StringIndexOutOfBoundsException e) {
+            newSeriesName = entry.getValue() + " " + "(ID: " + entry.getKey() + ")";
+          }
           logger.debug(String.format("Repeated series title \"%s\" found, changing to \"%s\" for admin-ui display",
               entry.getValue(), newSeriesName));
           list.put(entry.getKey(), newSeriesName);

